newrelic/packages/isort/stdlibs/py36.py

```diff
@@ -0,0 +1,224 @@
+"""
```

Comment:
Do we need the py36 stdlib if the agent only supports 3.7+?

Reply:
No we don't but removing it requires modifying the __init__.py file from the latest on isort so I thought it was simpler to just leave it in for copy-paste purposes. I don't have any strong opinions on it though.